test/src/test/java/com/coderanch/test/TestTest.java

```diff
@@ -0,0 +1,51 @@
+package com.coderanch.test;
+
+import org.junit.Test;
+
+public final class TestTest {
+
+    private final ConsistentComparableTest testToTest = new ConsistentComparableTest();
+
+    @Test
+    public void allTests_withConsistentObjects_succeed() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this method name to match the regular expression: '^test[A-Z][a-zA-Z0-9]*$' ([squid:S3578](https://sonarcloud.io/organizations/coderanch-corral/rules#rule_key=squid%3AS3578))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.coderanch%3Ablackjack-parent&pullRequest=11&issues=AWbLCXsMSgN4uCclysow&open=AWbLCXsMSgN4uCclysow)

[ISSUE_KEY]: AWbLCXsMSgN4uCclysow